README.md

```diff
@@ -1,3 +1,19 @@
+---
```

Comment:
Need this HR?

Reply:
I think we should keep this especially if we are moving it below the first section - so it's clear its kind of separate from the readme.